src/main/kotlin/no/nav/syfo/kafka/producers/tms_microfrontend/MicrofrontendKafkaProducer.kt

```diff
@@ -0,0 +1,30 @@
+package no.nav.syfo.kafka.producers.tms_microfrontend
+
+import no.nav.syfo.Environment
+import no.nav.syfo.kafka.common.JacksonKafkaSerializer
+import no.nav.syfo.kafka.common.producerProperties
+import no.nav.syfo.kafka.common.topicMinSideMicrofrontend
+import org.apache.kafka.clients.producer.KafkaProducer
+import org.apache.kafka.clients.producer.ProducerConfig
+import org.apache.kafka.clients.producer.ProducerRecord
+
+class MicrofrontendKafkaProducer(
+    val env: Environment
+) {
+    private val kafkaConfig = producerProperties(env).apply {
+        put(ProducerConfig.KEY_SERIALIZER_CLASS_CONFIG, "org.apache.kafka.common.serialization.StringSerializer")
+        put(ProducerConfig.VALUE_SERIALIZER_CLASS_CONFIG, JacksonKafkaSerializer::class.java)
+    }
+
+    private val kafkaProducer = KafkaProducer<String, MinSideRecord>(kafkaConfig)
+
+    fun sendRecordToMFSelector(minSideRecord: MinSideRecord) {
```

Comment:
Jeg tror at det er mer lesbar og forståelig å forholde seg til hele ord i stedet for forkortelser (bortsett fra helt vanlige begrep som DB). Betyr "MF" _MicroFrontend_ her?

Reply:
@Vitenok Det stemmer. Jeg kan godt ekspandere til MicroFrontend i stedet for MF der det forkortes 👍

Reply:
Fikset